exec/vm.go

```diff
@@ -160,6 +196,12 @@ func NewVM(module *wasm.Module) (*VM, error) {
 	return &vm, nil
 }
 
+// NewVM creates a new VM from a given module. If the module defines a
+// start function, it will be executed.
+func NewVM(module *wasm.Module) (*VM, error) {
```

Comment:
as commented on the previous incarnation of this PR, I think it's fair game to just call the function you name `NewVMWithOptions` simply `NewVM`.